kube-derive/tests/crd_schema_test.rs

```diff
@@ -208,21 +208,17 @@ fn test_crd_schema_matches_expected() {
                                             },
 
                                             "nullableSkipped": {
-                                                "nullable": true,
```

Comment:
hm, while this makes the tests pass, it is kind of hamstringing the readability of the tests a bit.
maybe call these optionWithDefault, optionSkipped, optionPlain, optionSkippedWithDefault instead

Reply:
thanks, I should have paid more attention, changed it.